oomph-plugin-notes.js

```diff
@@ -0,0 +1,76 @@
+(function($) {
+	$('.plugin-notes-container.editable, .plugin-notes-edit').on('click', function() {
+		var $container = $(this).closest('.plugin-notes-container')
+		  , $textarea = $('<textarea class="widefat">')
+		  , $saveButton = $('<button class="button-primary">')
+		  , $cancelButton = $('<button class="button-secondary">')
+		;
+
+		if($container.hasClass('editing')) {
+			return;
+		}
+
+		$container.addClass('editing');
+
+		$saveButton.text(OPN.text.save);
+		$cancelButton.text(OPN.text.cancel);
+
+		$container.data('plugin-markup', $container.html());
+
+		$textarea.val($container.data('plugin-notes'));
+
+		function adjust() {
+			$textarea.height(1);
+			$textarea.height($textarea[0].scrollHeight);
+		}
+
+		// Make sure the text area is the same number of lines contained within
+		$textarea.on('keyup', adjust);
+
+		$container.html('');
+		$container.append($textarea);
```

Comment:
Doesn't make much difference, but you could chain these calls OR you can actually pass all of the elements in one `append()` call (it accepts multiple args).